ironfish-cli/src/commands/wallet/multisig/ledger/restore.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../../command'
+import * as ui from '../../../../ui'
+import { Ledger } from '../../../../utils/ledger'
+
+export class MultisigLedgerRestore extends IronfishCommand {
+  static description = `Restore encrypted multisig keys to a Ledger device`
```

Comment:
Same comment as the other PR, but I think the phrasing is fine here
```suggestion
  static description = `restore encrypted multisig keys to a Ledger device`
```